.coveragerc

```diff
@@ -425,6 +425,7 @@ omit =
     homeassistant/components/foscam/camera.py
     homeassistant/components/foursquare/*
     homeassistant/components/free_mobile/notify.py
+    homeassistant/components/freebox/alarm_control_panel.py
```

Comment:
Why have you added this file here ?
Non tested files should be here, and you added a test !

Reply:
```suggestion
```